src/components/Tabs/styles.css

```diff
@@ -69,11 +69,13 @@
 }
 
 .activeDropdownTabClass {
-  margin: 0;
-  padding-right: 1.5rem;
+  margin: 0 1.5rem 0 0;
+  padding-left: 1.5rem;
 }
 
 .activeDropdownTabWrapper {
   display: flex;
   align-items: center;
+  justify-content: flex-start;
+  width: 142px;
```

Comment:
This fixed width is not gonna work with all cases(not for cases where the widest tab got width less than 142px), this why it should be set dynamically to the width of the widest tab in the tabs list.

Thinking about adding a class name prop to `TabDropdownTrigger` to be able set the width dynamically, what do you think ?

Reply:
So found an improvement, instead of using static width for the dropdown trigger, I'm using a `ref` of the dropdown list element, and updating the dropdown trigger width dynamically

Reply:
There is still one thing to fix/improve:

currently the trigger width is re-calculated each time the dropdown list is toggled - new `ref` is created and the width is recalculated. 
Which causes the dropdown to `jump` on tab select, this can be fixed by calculating the width once and not on each tab select

Reply:
Resolved.
inspired by: https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node